3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,69 @@
+//Flight booking
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (useFormalName == true && gender == "man") {
+    return `Lord ${firstname} ${surname}`;
+  } else if (useFormalName == true && gender == "woman") {
+    return `Miss ${firstname} ${surname}`;
+  } else {
+    return `${firstname} ${surname}`;
+  }
+}
+
+let fullname1 = getFullname("Sevda", "Naziri", true, "woman");
+let fullname2 = getFullname("John", "Wick", true, "man");
+
```

Comment:
🚀 Great job on your implementation!
🌟 Your code handles formal names effectively, and that's fantastic! 
💡 To enhance it further, consider using the strict equality operator (`===`) for `useFormalName` for added clarity.💡 It's better to use `const` instead of `let` if you are not going to reassign a new value to the variable